### PR TITLE
Distance

### PR DESCRIPTION
**Pull request by raokrutarth**

### CODE REVIEW

**Review by mpace965 — APPROVED** [review]
Looks good to me, tested on my machine

**Review by mpace965 — CHANGES_REQUESTED** [review]
The dataset needs to be put in and accessed from static storage.

**Review by TomFanella4 — CHANGES_REQUESTED** [review]
Can we put the changes to the dataset on hold? Making the large files even larger is going to make pruning take longer.

**Review by TomFanella4 — APPROVED** [review]

**Comment by raokrutarth** [comment]
@mpace965 is static storage google drive? I believe I'm writing the generation code as if we use the files directly from disk and Tom will transition that code to the cloud file access and clean up any large files in data/

File: backend/Handlers.go
Comment (mpace965) [inline backend/Handlers.go]: `seedUser` can be removed in its entirety

File: backend/data/model.go
Comment (mpace965) [inline backend/data/model.go:176]: What is the purpose of this file?

**Review by mpace965 — APPROVED** [review]
I had a couple of questions, but otherwise looks good for now.

**Review by mpace965 — CHANGES_REQUESTED** [review]
This branch does not currently build. I get this when running `go build`:

```
# meetover/backend
./Handlers.go:90:37: undefined: cachedUsers
./Handlers.go:92:21: undefined: cachedUsers
./Handlers.go:110:35: unknown field 'Long' in struct literal of type Geolocation
./Handlers.go:110:50: unknown field 'Lat' in struct literal of type Geolocation
./Location.go:37:25: center.Lat undefined (type Geolocation has no field or method Lat)
./Location.go:37:37: center.Long undefined (type Geolocation has no field or method Long)
./Location.go:37:49: point.Lat undefined (type Geolocation has no field or method Lat)
./Location.go:37:60: point.Long undefined (type Geolocation has no field or method Long)
./UserManagement.go:252:2: undefined: cachedUsers
```

**Comment by raokrutarth** [comment]
My bad on the build issue. Was not aware of the geoloc field name change.

File: backend/UserManagement.go
Comment (TomFanella4) [inline backend/UserManagement.go]: Why do we still need `cachedUsers`? I thought we decided we would get all users from Firebase?

File: backend/UserManagement.go
Comment (TomFanella4) [inline backend/UserManagement.go]: Related to `cachedUsers` comment.

File: backend/main.go
Comment (TomFanella4) [inline backend/main.go]: Related to `cachedUsers` comment.

File: backend/Handlers.go
Comment (TomFanella4) [inline backend/Handlers.go]: Related to `cachedUsers` comment.

File: backend/Location.go
Comment (TomFanella4) [inline backend/Location.go]: `makeTimestamp()` can be removed because the timestamp is created on the client.

File: backend/data/model.go
Comment (TomFanella4) [inline backend/data/model.go:176]: I'm also confused as to why we need this file.

**Review by TomFanella4 — CHANGES_REQUESTED** [review]
Branch now builds. Left several comments that need to be addressed before merged.

**Review by TomFanella4 — APPROVED** [review]

**Pull request merged**
